nkms_eth/project/contracts/MinersEscrow.sol

```diff
@@ -638,7 +640,7 @@ contract MinersEscrow is Issuer {
             bytes32(uint8(MinerInfoField.LastActivePeriod)), miner, 0)) == info.lastActivePeriod);
         require(uint256(delegateGet(_testTarget, "getMinerInfo(uint8,address,uint256)",
             bytes32(uint8(MinerInfoField.DowntimeLength)), miner, 0)) == info.downtime.length);
-        for (i = 0; i < info.downtime.length && i < MAX_PERIODS; i++) {
+        for (i = 0; i < info.downtime.length && i < 10; i++) {
```

Comment:
Wasn't this better as a constant?

Reply:
Incorrect use of this constant, I'll add another one

Reply:
:+1: